### PR TITLE
refactor(Flow): match Figma and utilize Tailwind

### PR DESCRIPTION
**Pull request by justinhough**
**Related Issue:** #2467 (relates to #2473)

## Summary

For `calcite-flow` and `calcite-panel`, fixed spacing issues in relation to back button width and space around header / summary area.

- Back button changed scale to "m" from "s"
- Header/summary area changed padding to match Figma `py-5 (20px)` and `py-4 (16px)`

### CODE REVIEW

**Comment by julio8a** [comment]
@asangma @bstifle can you guys review this one? Screener too

**Comment by bstifle** [comment]
@justinhough alrighty, so the only thing im seeing that needs to be fixed is the border colors.

The bottom border and border separating the back button are using rgb(234,234,234), which comes out to #EAEAEA

Can we change those borders to use ui-border-2 or equivalent? rgba(212,212,212)  #D4D4D4

code
image
figma
image

**Review by bstifle — APPROVED** [review]
Noice!

Ship 🚢

File: src/components/calcite-panel/calcite-panel.scss
Comment (caripizza) [inline src/components/calcite-panel/calcite-panel.scss:92]: Can you remove the duplicate full width style within this class?

(@julio8a I'd love to resurrect the border-side-style plugin I had added a while back but then was directed to remove. [It would](https://github.com/Esri/calcite-components/blob/49c0adfd3971b140e7de8ce1c8271513c4883825/tailwind.config.js#L263) allow us to style borders like `@apply border-l border-l-solid border-l-color-input;` in the near future.)

**Review by caripizza — APPROVED** [review]
LGTM! Just a tiny cleanup request (on code you didn't touch - thanks for helping tidy it up).

File: src/components/calcite-panel/calcite-panel.scss
Comment (julio8a) [reply src/components/calcite-panel/calcite-panel.scss:92]: @caripizza, let's discuss in our next core team meeting.

**Review by julio8a — APPROVED** [review]
Nice!

**Comment by justinhough** [comment]
@caripizza Do I need to merge `master` into this before merging this PR?

**Comment by caripizza** [comment]
>@caripizza Do I need to merge master into this before merging this PR?

@justinhough yes please! Let's keep these refactor prs nice and up to date with master for Screener changes.

**Comment by justinhough** [comment]
@caripizza Sounds great. Just pushed up the merge of master after testing it locally.

**Pull request merged**
